autogen/agentchat/conversable_agent.py

```diff
@@ -42,7 +44,7 @@ def colored(x, *args, **kwargs):
 F = TypeVar("F", bound=Callable[..., Any])
 
 
-class ConversableAgent(Agent):
+class ConversableAgent(LLMAgent, Agent):
```

Comment:
I remember that one design decision we made earlier was to avoid multi-inheritance. Is it because we are now using the "Protocol" design?

Reply:
Protocol inheritance is different from class inheritance since it only declares the class is implementing one or more protocols. This information is used by static type checkers only and does not affect runtime behaviour.